xbmc/interfaces/Builtins.cpp

```diff
@@ -1851,6 +1852,10 @@ int CBuiltins::Execute(const std::string& execString)
   {
     g_application.StopPVRManager();
   }
+  else if (execute == "pvr.searchmissingchannelicons")
+  {
+    g_application.SearchMissingChannelIcons();
```

Comment:
Please use the async method `TriggerSearchMissingChannelIcons` which do not block the further processing here. And IMO there is no need to log anything if the manager is not yet started. So doing the following is enough.

```
g_PVRManager.TriggerSearchMissingChannelIcons();
```